packages/zowe-explorer/webpack.config.js

```diff
@@ -50,6 +50,9 @@ const config = {
     resolve: {
         // support reading TypeScript and JavaScript files, 📖 -> https://github.com/TypeStrong/ts-loader
         extensions: [".ts", ".js"],
+        alias: {
+            "@zowe/zowe-explorer-api$": path.resolve(__dirname, "..", "zowe-explorer-api/src"),
```

Comment:
Does this alias allow the incremental builds or just including in bundle when resolving? i'm curious about what this does :)

Reply:
This isn't needed for the incremental builds, but is one of 2 changes I had to make for breakpoints to work:
* The `resolve.alias` tells Webpack to reference TS files rather than JS files for any imports from the "@zowe/zowe-explorer-api" package, which allows us to set breakpoints directly in the TS files 🙂 
* The `devtoolModuleFilenameTemplate` needs to use absolute paths, because Webpack doesn't evaluate relative paths correctly for source files outside of the directory where webpack.config.js resides